futurenet/soroban-rpc/etc/stellar-captive-core.cfg

```diff
@@ -13,6 +13,7 @@ ENABLE_SOROBAN_DIAGNOSTIC_EVENTS=__ENABLE_SOROBAN_DIAGNOSTIC_EVENTS__
 
 FAILURE_SAFETY=0
 UNSAFE_QUORUM=true
+LIMIT_TX_QUEUE_SOURCE_ACCOUNT=true
```

Comment:
just confirming usage of the flag, is it required for captive core instances? since no tx's are submitted to that type of instance.

Reply:
no ? I'm pretty sure that the soroban-rpc is submitting transaction using it's own captive core instace.

Reply:
I see, my apology, `stellar-core-url` will default to the localhost captive core url if is not set otherwise, ok, thanks for clarification!